packages/workers-shared/router-worker/src/index.ts

```diff
@@ -1,20 +1,24 @@
+import type { UnstableCanFetchMethod } from "../../asset-worker/src";
 import type { RoutingConfig } from "../../utils/utils";
 
 interface Env {
-	ASSET_WORKER: Fetcher;
+	ASSET_WORKER: Fetcher & {
+		unstable_canFetch: UnstableCanFetchMethod;
```

Comment:
how does this work in `wrangler dev`?

Reply:
the asset plugin sets up a service binding from RW to AW - seems to work fine!